cupsfilters/image-jpeg-xl.c

```diff
@@ -28,12 +27,14 @@ is_jpegxl(const unsigned char *header, size_t len)
 }
 
 /*
- * cf_image_create_from_jxl_decoder() - Create a cf_image_t from a libjxl decoder.
+ * cfImageCreateFromJxlDecoder() - Create a cf_image_t from a libjxl decoder.
```

Comment:
Why are you changing the name in the comment?

The scheme for the name, underscore-separated lowercase is correct, as the function is static.